start.py

```diff
@@ -138,6 +150,8 @@ def start():
             compose_files.append(path_mapping["django_server"])
         else:
             compose_files.append(path_mapping[args.mode])
+        if args.__dict__["elastic"] or int(os.environ.get("LOCAL_ELASTICSEARCH")):
+            compose_files.append(path_mapping["elastic"])
```

Comment:
there is no need to use `LOCAL_ELASTICSEARCH` at all. It is just enough the command `--elastic`. This variable does not help in anyway. On the contrary it creates confusion. Why 2 different ways to configure the same thing?

Reply:
Alright. Will remove it.